### PR TITLE
Add MiniValine User Guide

### PR DESCRIPTION
**Pull request by MHuiG**
This is the documentation for [MiniValine](https://github.com/theme-next/awesome-next/pull/21)

https://github.com/theme-next/awesome-next/pull/21

This information can make users better use it

### CODE REVIEW

**Review by stevenjoezhang — APPROVED** [review]

File: source/docs/third-party-services/comments.md [hunk 1]
```diff
@@ -116,6 +116,48 @@ Valine is a fast, simple & efficient Leancloud based no back end comment system.
       serverURLs: # When the custom domain name is enabled, fill it in here (it will be detected automatically by default, no need to fill in)
     ```
 
+### MiniValine (China Modified from Valine)
```
Comment (sli1989) [inline source/docs/third-party-services/comments.md]: hi, can you just use MiniValine as title to make hyperlinks working. `/docs/third-party-services/comments#MiniValine`
as well as Valine (China), Changyan (China) to Valine, Changyan

**Review by sli1989 — CHANGES_REQUESTED** [review]

File: source/docs/third-party-services/comments.md (hunk 1 above)
Comment (sli1989) [reply source/docs/third-party-services/comments.md]: oh, no, I mean changing the titles in `/docs/third-party-services/comments.md`.
the title must be same as anchor.

Comment (sli1989) [reply source/docs/third-party-services/comments.md]: well, i saw the second commit. em, that would work too.

Comment (MHuiG) [reply source/docs/third-party-services/comments.md]: https://5e81b47f200691000a5bd29d--theme-next.netlify.com/docs/third-party-services/

It has been confirmed that anchor links seem to be working properly.

Comment (sli1989) [reply source/docs/third-party-services/comments.md]: but it seems better for explation (china) in index.md. well, forget it.

**Review by sli1989 — APPROVED** [review]

File: source/docs/third-party-services/index.md [hunk 2]
```diff
@@ -16,9 +16,9 @@ In any time you can extend needed functions by using third-party services suppor
 * [DisqusJS](/docs/third-party-services/comments#DisqusJS)
 * [LiveRe](/docs/third-party-services/comments#LiveRe)
 * [Gitalk](/docs/third-party-services/comments#Gitalk)
-* [Valine](/docs/third-party-services/comments#Valine)
-* [Changyan](/docs/third-party-services/comments#Changyan)
-* [MiniValine](/docs/third-party-services/comments#MiniValine)
+* [Valine (China)](/docs/third-party-services/comments#Valine-China)
```
Comment (sli1989) [inline source/docs/third-party-services/index.md:19]: hi, i mean this.

```
* [Valine (China)](/docs/third-party-services/comments#Valine)
* [Changyan (China)](/docs/third-party-services/comments#Changyan)
* [MiniValine (China)](/docs/third-party-services/comments#MiniValine)
```

File: source/docs/third-party-services/comments.md [hunk 3]
```diff
@@ -86,7 +86,7 @@ Hexo has encode URLs by default since version 4.0: [PR 3708](https://github.com/
 Gitalk relies on the md5 value of the page path for tagging, and this may cause issues. It's recommended to upgrade Hexo to 4.0 (or above), NexT to 7.7.0 (or above) to use Gitalk.
 {% endnote %}
 
-### Valine
+### Valine (China)
```
Comment (sli1989) [inline source/docs/third-party-services/comments.md]: NO china in `/docs/third-party-services/comments.md`.

Comment (MHuiG) [reply source/docs/third-party-services/comments.md]: image

I made a change following the example below and the anchor link is working.

https://deploy-preview-90--theme-next.netlify.com/docs/third-party-services/

Comment (MHuiG) [reply source/docs/third-party-services/comments.md]: Parentheses need to be represented as "-" at anchor links

Comment (sli1989) [reply source/docs/third-party-services/comments.md]: oo, thanks. u are right.

Comment (MHuiG) [reply source/docs/third-party-services/comments.md]: You're welcome.

**Comment by MHuiG** [comment]
It's been too long, I will close this PR

**Comment by MHuiG** [comment]
It's been too long, I will close this PR

**Pull request closed without merging**